core/src/types/voting_power.rs

```diff
@@ -4,9 +4,61 @@ use std::iter::Sum;
 use std::ops::{Add, AddAssign};
 
 use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use ethabi::ethereum_types as ethereum;
 use eyre::{eyre, Result};
 use num_rational::Ratio;
 
+/// Namada voting power, normalized to the range `0 - 2^32`.
```

Comment:
How many times have we implemented this type :laughing:

Reply:
no idea. I didn't find it in our code, so I made my own type lmao